ra/ra.go

```diff
@@ -1418,7 +1434,7 @@ func (ra *RegistrationAuthorityImpl) issueCertificateInner(
 		return nil, nil, wrapError(err, "parsing final certificate")
 	}
 
-	ra.countCertificateIssued(ctx, int64(acctID), parsedCertificate.DNSNames)
+	ra.countCertificateIssued(ctx, int64(acctID), parsedCertificate.DNSNames, isRenewal)
```

Comment:
Since this is still "best effort" and never resulting in a user-surfaced error, why not do this in a goroutine?